src/holodeck/sensors.py

```diff
@@ -520,6 +520,39 @@ def data_shape(self):
         return [1]
 
 
+class RangeFinderSensor(HolodeckSensor):
+    """Returns collision distances in directions specified by parameters.
+
+    **Configuration**
+
+    The ``configuration`` block (see :ref:`configuration-block`) accepts the 
+    following options:
+
+    - ``LazerMaxDistance``: Max Distance in meters of RangeFinder. (default 10)
+    - ``LazerCount``: Number of lazers in sensor. (default 1)
+    - ``LazerAngle``: Angle of lazers from origin. Measured in degrees. Positive angles point up. (default 0)
+    - ``LazerDebug``: Show debug traces. (default false)
```

Comment:
I feel like this sensor would benefit from having some screenshots to show how it works and how the angling works. We don't have a place in the normal docs though for sensors, so I'd say either see if you can embed screenshots here in the docstring or find a logical place to put it in the doc tree and link to it from here.